src/components/vr_module/src/commands/support_service.cc

```diff
@@ -77,7 +77,11 @@ void SupportService::on_event(
   vr_hmi_api::SupportServiceResponse response;
   if (message.has_params() && response.ParseFromString(message.params())) {
     bool supported = response.result() == vr_hmi_api::SUCCESS;
-    module_->set_supported(supported);
+    if (supported) {
```

Comment:
The shortest code is not always readable code.
Ternary operator is good to use for assign variable in my opinion.